Tasks/DownloadPackageV0/task.json

```diff
@@ -9,7 +9,7 @@
     "version": {
         "Major": 0,
         "Minor": 1,
-        "Patch": 17
+        "Patch": 18
```

Comment:
Everywhere else we follow milestone number for minor version. Should we change this?